src/util/sqs.js

```diff
@@ -0,0 +1,46 @@
+const assert = require('assert');
```

Comment:
Exact copy from `stats-collector` only change is we are now passing through the `call` function